cypress/support/step_definition/management/admin_management/administrateExternalToolsSteps.spec.js

```diff
@@ -0,0 +1,102 @@
+import Management from '../../../pages/management/pageManagement'
+import Teams_Common from '../../../pages/teams/pageCommonTeams'
+
+const management = new Management()
+const teamCommon = new Teams_Common()
+
+//Scenario: Deactivate Chat
+//Given I am logged in as a 'admin' at 'brb'
+//step defined --> \step_definition\commonGlobalSteps\loginLogoutSteps.spec.js
+
+//When I go to administration page
+//step defined --> \step_definition\management\CommonManagementRelatedSteps.spec.js
+
+//And I go to school administration
+//step defined --> \step_definition\management\CommonManagementRelatedSteps.spec.js
+
+//And I go to new school administration pag
+//step defined --> \step_definition\management\CommonManagementRelatedSteps.spec.js
+
+When('I click toggle switch to deactivate the chat', () => {
+  management.clickChatToggleSwitch()
+})
+
+//And I click save general settings button
```

Comment:
Question: These commented steps are specific to the feature or we have to add them in all step definitions the same steps? Asked becuase I saw lots of steps which I don't know why they're present here. 

Suggestion: Just mention the steps which are relevant to the feature and guide the reader that if you are looking to some particular step related to this feature then please go to that file. Otherwise it would be a big mess in the end to handle. Also, already mentioned in another PR that commented step should be presented at the top and the narration of path of the step should be concise i.e. some table in which all steps should be shown and the single path where they will be available.

When (xyz) ----|
When (abc) ----|______ step_definition/commonxyz/

Reply:
- the paths are defined already in line 25, 28, 31, 34, 37 ...etc where the location of the each step definition is defined.
- so, every step-definition is followed by the user flow exactly as defined in the feature file, if they already defined before they are part of common file and location is mentioned as comment.
- if we mention already defined steps location at the top then again its not consistent with the feature file user journey flow.

I would say we can keep it this way in the first version and can be adapted later because there is another input from the dev team so we need gather and align it in the next version of it.